### PR TITLE
fastfetch: Add version 1.8.2

### LINKED ISSUE #10152: [Request]: fastfetch
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

fastfetch

### Description

Like neofetch, but much faster because written in C.

### Homepage

https://github.com/LinusDierheimer/fastfetch

### Download Link(s)

https://github.com/LinusDierheimer/fastfetch/releases/download/1.8.2/fastfetch-1.8.2-Windows.zip

### Some Indication of Popularity/Repute

478 stars and 56 forks

### PR DESCRIPTION
**Pull request by CarterLi**
Closes #10152

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/fastfetch.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.8.2",
+    "description": "Like neofetch, but much faster because written in C",
+    "homepage": "https://github.com/LinusDierheimer/fastfetch",
+    "license": "Freeware",
```
Comment (aliesbelik) [inline bucket/fastfetch.json:5]: Please change to `MIT` identifier (should be a valid [SPDX Identifier](https://spdx.org/licenses) as per Contributing Guide).

**Comment by CarterLi** [comment]
Done

**Comment by CarterLi** [comment]
Closing in favor of https://github.com/ScoopInstaller/Main/pull/4334

**Comment by sitiom** [comment]
Oops, didn't see this, but fastfetch has reached its 500-star mark so hopefully it's okay 😅

**Pull request closed without merging**